.github/workflows/publish.yaml

```diff
@@ -29,9 +29,9 @@ jobs:
           distribution: 'adopt'
           cache: 'maven'
           # Configure ~/.m2/settings.xml
-          server-id: ossrh
-          server-username: OSSRH_USERNAME
-          server-password: OSSRH_PASSWORD
+          server-id: central
+          server-username: CP_USERNAME
+          server-password: CP_PASSWORD
```

Comment:
I think these need a matching change down on line 53 to load the configured workflow creds into the env vars.

Reply:
Thanks for pointing this out, I forgot to update them. Fixed in my latest commit.